src/vs/editor/common/editorCommon.ts

```diff
@@ -4164,6 +4167,33 @@ export var EventType = {
 };
 
 /**
+ * Positions on the view
+ */
+export const ViewPosition = {
+	LineStart: 'lineStart',
+	LineFirstNonWhitespaceCharacter: 'lineFirstNonWhitespaceCharacter',
```

Comment:
Sometimes referred to as "soft start". @jrieken @sandy081 -- would `lineSoftStart` be easier to program against?